enums.d.ts

```diff
@@ -129,10 +129,10 @@ declare namespace RageEnums {
 	}
 
 	const enum VehicleSeat {
-		DRIVER = -1,
-		PASSENGER_1 = 0,
-		PASSENGER_2 = 1,
-		PASSENGER_3 = 2
+		DRIVER = 0,
+		PASSENGER_1 = 1,
+		PASSENGER_2 = 2,
+		PASSENGER_3 = 3
```

Comment:
Isnt this also applyable on the 0.3.7.2 version? By the way, aren't there more than 4 seats (including the driver)?

Reply:
By the way: You can write the enum short by writing this:
```ts
const enum Test {
    DRIVER, // would be equivalent to 0
    PASSENGER_1, // would be equivalent to 1
    PASSENGER_2 // and so on
}
```

Reply:
Well, passenger seats always vary for each vehicle and some have 6 or 8 (not sure which is the max though). What do you recommend?

Reply:
Current maximum amount is 16. So 16 passenger seats.